sdks/python/apache_beam/coders/coders.py

```diff
@@ -886,12 +907,14 @@ def __init__(self, wrapped_value_coder, window_coder=None):
     self.wrapped_value_coder = wrapped_value_coder
     self.timestamp_coder = TimestampCoder()
     self.window_coder = window_coder
+    self.pane_info_coder = PaneInfoCoder()
 
   def _create_impl(self):
     return coder_impl.WindowedValueCoderImpl(
         self.wrapped_value_coder.get_impl(),
         self.timestamp_coder.get_impl(),
-        self.window_coder.get_impl())
+        self.window_coder.get_impl(),
+        self.pane_info_coder.get_impl())
```

Comment:
Rather than introducing a PaneInfoCoder, just instantiate the PaneInfoCoderImpl in WindowedValueCoderImpl's constructor. (There's a TODO to remove the parameterizablility of the TimestampCoder as well.) Actually, we could consider just making {en,de}code_pane_info methods on WindowedValueCoder itself.